woocommerce/wc-admin
# Close activity panel tabs by default and track

Fixes #6449 

Closes the "Orders" activity panel by default and adds in tracking data about visible panels.

@pmcpinto This has some slight differences from the original issue because of where this information can be tracked, so please let me know if they look okay:
*  I renamed the tracks event to `wcadmin_activity_panel_visible_panels`.
* The current data shows what panels are visible, but not necessarily which are hidden. (`{ orders-panel: true }` and not `{ orders-panel: true, inventory-panel: false }`  We'll have to rework the logic in the panels a bit to show all, but not a big deal if this is important.
* This same event includes `taskList` which will be true or false depending on whether or not the task list is shown.

### Screenshots

<img width="857" alt="Screen Shot 2021-03-10 at 4 47 51 PM" src="https://user-images.githubusercontent.com/10561050/110702325-600b7680-81c0-11eb-91fc-5a654aa95e51.png">
<img width="831" alt="Screen Shot 2021-03-10 at 4 48 48 PM" src="https://user-images.githubusercontent.com/10561050/110702498-9648f600-81c0-11eb-8692-63928080d750.png">


### Detailed test instructions:

1. Open your browser console and enter `localStorage.setItem( 'debug', 'wc-admin:tracks' );`.
2. With the task list enabled, navigate to the homescreen.
3. Check that the `wcadmin_activity_panel_visible_panels` event is shown with `taskList: true` in its data.
4. Hide the task list.
5. Note that `wcadmin_activity_panel_visible_panels` event is shown with visible activity panels.
6. After refreshing, make sure that the "Orders" activity panel is closed by default.
